model/client4.go

```diff
@@ -2590,6 +2594,18 @@ func (c *Client4) GetAudits(page int, perPage int, etag string) (Audits, *Respon
 	}
 }
 
+// Admin Section
+
+// GetBrandImage retrieves the previously uploaded brand image.
+func (c *Client4) AdminUpdateUser(userId string, authData string, authService string, password string) (*User, *Response) {
+	if r, err := c.DoApiPost(c.GetAdminRoute()+"/users/update", fmt.Sprintf("{\"id\":\"%v\",\"auth_data\":\"%v\",\"auth_service\":\"%v\",\"password\":\"%v\"}", userId, authData, authService, password)); err != nil {
```

Comment:
Would be cleaner to use a struct or to create a map and convert that that to JSON rather than manually constructing the JSON